package.json

```diff
@@ -9,7 +9,8 @@
     "test": "jest",
     "test-coverage": "jest --coverage",
     "test-watch": "jest --watch",
-    "test-watch-silent": "jest --watch --silent"
+    "test-watch-silent": "jest --watch --silent",
+    "lint": "tslint --project ."
```

Comment:
We need command which fixes lint issues and also command for use prettier
"lint-and-fix": "eslint . --ext .ts --fix",
"prettier-format": "prettier --config .prettierrc **/*.ts --write",